TextEditor.h

```diff
@@ -227,11 +227,7 @@ class TextEditor
 	class ExternalUndoBufferInterface
 	{
 	public:
-		ExternalUndoBufferInterface() {};
-		~ExternalUndoBufferInterface() {};
-
-		virtual void AddUndo(UndoRecord&, TextEditor&)=0;
-

```

Comment:
I removed that useless lines as this interface is completely abstract a need no constructor/destructor.